src/includes/home/about.md

```diff
@@ -135,6 +135,8 @@ An extension is a simple collection of files that modify the browser’s appeara
 
 Add toolbar buttons, menu choices, and—only in Firefox—sidebars to display additional content. Manage tab behavior and create pop-up windows that respond to user events.
 
+[Learn more](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface)
```

Comment:
Could you please use locale-less URLs? (and yes, we are probably not doing that everywhere)

```suggestion
[Learn more](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/user_interface)
```